src/tracer/Tracee.cc

```diff
@@ -68,47 +71,65 @@ reg_t Tracee::getSyscallNumber() {
 reg_t Tracee::getSyscallArgument(uint8_t argumentNumber) {
 #if defined(__x86_64__)
     if (syscallArch_ == Arch::X86) {
-        if (argumentNumber == 0)
+        if (argumentNumber == 0) {
```

Comment:
a może by to switchem, albo jakąś tablicą wskaźników na pola?

Reply:
Można, ale ja tego kodu nie chciałem całego poprawiać...

Reply:
Poprawię na switcha